frontend/packages/console-shared/src/components/multi-tab-list/MultiTabListPage.tsx

```diff
@@ -31,9 +31,12 @@ const MultiTabListPage: React.FC<MultiTabListPageProps> = ({
   } = match;
   const onSelectCreateAction = (actionName: string): void => {
     const selectedMenuItem: MenuAction = menuActions[actionName];
-    const namespace = selectedMenuItem.model.namespaced ? ns || 'default' : ns;
-    const modelRef = referenceForModel(selectedMenuItem.model);
-    let url = namespace ? `/k8s/ns/${namespace}/${modelRef}/~new` : `/k8s/cluster/${modelRef}/~new`;
+    let url: string;
+    if (selectedMenuItem.model) {
+      const namespace = selectedMenuItem.model.namespaced ? ns || 'default' : ns;
```

Comment:
I feel that this check is redundant. WDYT?
```suggestion
      const namespace = ns || 'default';
```